### PR TITLE
Update teamviewer from 15.26.4 to 15.26.5

### PR DESCRIPTION
**Pull request by owine**
Created with `brew bump-cask-pr`.

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

**Comment by miccal** [comment]
Ping @carlocab.

The CI tests for Catalina are now failing, which is to be expected, but this is a way to deal with the issues raised in https://github.com/Homebrew/homebrew-cask/issues/118316.

**Review by carlocab — DISMISSED** [review]
Makes sense to me, but we should find a way to avoid this failing in CI.

**Comment by miccal** [comment]
I am not sure how we could that -- can we somwhow make it so the CI tests are _not_ run on 10.15 and _only_ run on 11.0?

**Comment by carlocab** [comment]
A really dirty hack might be to do this instead:
```diff
diff --git a/Casks/teamviewer.rb b/Casks/teamviewer.rb
index 63adb4d948..2c048092bb 100644
--- a/Casks/teamviewer.rb
+++ b/Casks/teamviewer.rb
@@ -23,7 +23,11 @@ cask "teamviewer" do
 
   auto_updates true
   conflicts_with cask: "teamviewer-host"
-  depends_on macos: ">= :el_capitan"
+  if MacOS.version == :catalina
+    depends_on macos: ">= :big_sur"
+  else
+    depends_on macos: ">= :el_capitan"
+  end
 
   if MacOS.version <= :high_sierra
     pkg "Install TeamViewer.pkg"
```
Might still fail CI though.

Otherwise we can probably modify our runner-selection script, but I'd like to keep any hacks out of there ideally.

**Comment by miccal** [comment]
@carlocab how about this?

File: Casks/teamviewer.rb [hunk 1]
```diff
@@ -27,28 +27,37 @@
 
   if MacOS.version <= :high_sierra
     pkg "Install TeamViewer.pkg"
+  elsif MacOS.version == :catalina
+    # This Cask should be installed and uninstalled manually on Catalina.
+    # See https://github.com/Homebrew/homebrew-cask/issues/76829
+    installer manual: "Install TeamViewer.app/Contents/Resources/Install TeamViewer.pkg"
```
Comment (carlocab) [inline Casks/teamviewer.rb:33]: Much better! Thanks, @miccal

**Review by carlocab — APPROVED** [review]

**Pull request merged**